docs/development/setup-recommended.md

```diff
@@ -317,7 +317,22 @@ cd zulip
 vagrant up --provider=docker
 ```
 
-The first time you run this command it will take some time because vagrant
+If you are using macOS, you may see something like this:
+```bash
+The provider 'docker' that was requested to back the machine
+'default' is reporting that it is not usable on this system.
+```
+This likely means that Docker is not currently running. You can manually open Docker before starting the development environment: 
+
+```bash
+# On macOS or Linux:
+open a- Docker
```

Comment:
If [this copy of the `open(1)` manpage is correct](https://ss64.com/osx/open.html), it appears this incantation should be `open -a` (just a small typo!). I don't personally have a Mac to confirm.

Anyway, on Linux, this doesn't work: we use `systemctl enable --now docker.service` or `rc-add docker default && rc-service docker start` or so on, so Linux should be excluded from this comment.